docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -9,24 +9,50 @@ description: Managing your review workflows and stages in Strapi.
-- Review Workflows are only available with the Enterprise Edition. If for any reason your project was downgraded to the Community Edition, all review workflows data would not be deleted but kept until the project goes back to Enterprise.
-:::
+<ThemedImage
+  alt="Workflow list view"
+  sources={{
+    light: '/img/assets/review-workflows/list-view-light.png',
+    dark: '/img/assets/review-workflows/list-view-dark.png',
+  }}
+/>
+
+In many organizations different teams review different parts of content. By using different review workflows for different content-types, it is possible to adjust each workflow to the needs of each team involved.
+
+The default workflow is configured to have 4 stages: To do, In progress, Ready to review, and Reviewed. All 4 stages can be edited, reordered or deleted as needed, and it is also possible to add new stages.
+
+Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content), review workflows must be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*. The Review workflows settings are only available to users with the Super Admin role by default. Other roles must be granted the **Review workflows** permissions. See [Users, Roles, & Permissions](/user-docs/users-roles-permissions) for more information.
+
+
+## Creating or editing a workflow
 
 <ThemedImage
-  alt="Review Workflows Settings"
+  alt="Workflow edit view"
   sources={{
-    light: '/img/assets/review-workflows/review-workflows.png',
-    dark: '/img/assets/review-workflows/review-workflows_DARK.png',
+    light: '/img/assets/review-workflows/edit-view-light.png',
+    dark: '/img/assets/review-workflows/edit-view-dark.png',
   }}
 />
 
-There are 4 stages in the default review workflow available in Strapi: To do, In progress, Ready to review, and Reviewed. All 4 stages can be edited, reordered or deleted as needed, and it is also possible to add new stages.
+1. Click on the **Create new workflow** button or on the edit button ![Edit icon](/img/assets/icons/edit.svg) of a workflow.
+2. In the workflow edit interface, configure the new workflow:
 
-## Adding a new stage
+    | Setting name   | Instructions                                                             |
+    | -------------- | ------------------------------------------------------------------------ |
+    | Name           | Write a unique name of workflow.                                         |
+    | Content-Types  | (optional) Assign this workflow to one or more existing content-types.   |
+    | Stages         | Add review stages (see [Adding a new stage](#adding-a-new-stage). |
+
+3. Click on the **Save** button. The new workflow will be displayed in the list-view and for every content-type assigned.
+
+:::note
+- The maximum number of [workflows and stages per workflow is limited](https://strapi.io/pricing-cloud).
```

Comment:
```suggestion
The maximum number of [workflows and stages per workflow is limited](https://strapi.io/pricing-cloud).
```
No need for the bullet point since there's only one point in the callout.